auth/password_authenticator.cc

```diff
@@ -119,7 +123,6 @@ future<> password_authenticator::migrate_legacy_metadata() const {
 }
 
 future<> password_authenticator::legacy_create_default_if_missing() {
```

Comment:
`maybe_create_default_password_with_retries` is also called during the `do_after_system_ready` flow, maybe it should be updated as well?

Similarly for `maybe_create_default_role_with_retries` in `standard_role_manager`.

Reply:
yes it is, but this is auth-v2 flow, since there is no v2->v1 downgrade, and we don't know how potential future v2->v3 would look like, we don't need to change it